utest.h

```diff
@@ -154,6 +154,27 @@
 #define UTEST_EXTERN extern
 #endif
 
+#ifdef _WIN32
```

Comment:
I only use _MSC_VER to detect windows - any reason why you used _WIN32 here?

Reply:
`_MSC_VER` only refers to MSVC which is the compiler causing the warnings, and supports these pragmas. gcc(mingw) also sets `_WIN32` (and needs io.h included) but shouldn't have the `#pragma` block.

Reply:
If use of non-msvc compilers on windows is an explicit non-goal of the project I guess `#ifdef _MSC_VER` is enough - no need for the nested `#if`. However, if you do intend to support gcc on windows I think it needs to be left the way it is.

Reply:
So given that I have no testing (and I don't think any way TO test) on Windows that is non MSVC, I'd rather remove it and just use _MSC_VER - I don't want people thinking that there is support when there definitely isn't.

Reply:
Agreed. Will fix.